app/Models/Extensions/AlbumBuilder.php

```diff
@@ -87,10 +93,57 @@ public function getModels($columns = ['*']): array
 
 			$maxTsSelect = $minTsSelect->clone()->select(DB::raw('MAX(taken_at)'));
 
-			$this->addSelect([
+			$selects = [
 				'min_taken_at' => $minTsSelect,
 				'max_taken_at' => $maxTsSelect,
-			]);
+			];
+
+			$showSubalbum = true;
+			$showSubalbumCount = false;
+			$showPhotoCount = false;
+			switch (Configs::getValueAsString('album_decorations')) {
```

Comment:
See comments in commits.

Reply:
In order to keep the complexity of queries to a minimum the back-end still considers the settings but now always returns the same set of attributes, i.e. `num_subalbums` and `num_photos`. Does that work?